lib/Common/Memory/HeapBlock.h

```diff
@@ -433,6 +433,7 @@ class SmallHeapBlockT : public HeapBlock
 #endif
     SmallHeapBlockBitVector* markBits;
     SmallHeapBlockBitVector  freeBits;
+    BVStatic<TBlockAttributes::PageCount * 0x1000 / sizeof(void*)> wbVerifyBits;
```

Comment:
>BVStatic<TBlockAttributes::PageCount * 0x1000 / sizeof(void*)> wbVerifyBits; [](start = 4, length = 76)

1. Can this be in "#if DBG"?
2. Why the magic number "0x1000"? Could you add a comment?